subsys/bluetooth/audio/shell/bap.c

```diff
@@ -752,6 +752,11 @@ static int lc3_release(struct bt_bap_stream *stream, struct bt_bap_ascs_rsp *rsp
 	return 0;
 }
 
+static struct bt_bap_unicast_server_register_param unicast_server_param = {
+	CONFIG_BT_ASCS_MAX_ASE_SNK_COUNT,
+	CONFIG_BT_ASCS_MAX_ASE_SRC_COUNT
+};
+
```

Comment:
You could technically move this to the `#if defined(CONFIG_BT_BAP_UNICAST_SERVER)` part of `cmd_init` as it is never used outside the scope of that function